src/sentry/relay/config.py

```diff
@@ -345,3 +349,70 @@ def _filter_option_to_config_setting(flt, setting):
                 # ret_val['options'] = setting.split(' ')
                 ret_val["options"] = list(setting)
     return ret_val
+
+
+# Note: names for extracted metrics start with `measurement.`, while the
+# in-event keys start with `measurements.`
+ALL_MEASUREMENT_METRICS = frozenset(
```

Comment:
in the ingest team we went back and forth a long time because we were unsure whether we want to enumerate all measurements that relay should process into metrics, or simply extract everything that the SDKs send. My intuition right now is that it's easy to update the server, and that we already have to do some additions ot the UI for every new measurement anyway.